src/components/HighlightContainer.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+import { colors } from "src/common/constants/colors";
+import { dropShadow } from "src/common/constants/dropShadow";
+import { CustomStyleProps } from "src/common/interface";
+
+const HighlightContainer = styled.div<CustomStyleProps>`
+    display: flex;
+    flex-direction: ${props => props.flexDirection || 'row'};
+    justify-content: ${props => props.justifyContent || 'center'};
+    align-items: ${props => props.alignItems || 'center'};
+    width: 900px;
+    height: 525px;
+    margin-left: 10%;
```

Comment:
Magic numbers

Reply:
Any suggestion on how should I define so many different widths and heights as constants?
Do percentages also need to be in constants?

Reply:
Please see suggestion in the other comment